profilegen/src/res/templates/module_template.template

```diff
@@ -0,0 +1,37 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+require '<%= @module_require %>'
+
+module Azure
+  module Management
+    module Profiles
+      module <%= @profile_name %>
+        module <%= @module_name %>
+          #
+          # Class for the <%= @class_name %> module
+          class <%= @class_name %>
+            attr_accessor <% @operation_types.each do |operation_type| %> :<%= operation_type[:operation_name] %>,<% end %> :model_types
+
+            def initialize(configurable, base_url, options)
+              client = <%= @management_client %>.new(configurable.credentials, base_url, options)
+              client.subscription_id = configurable.subscription_id
+              <%- @operation_types.each do |operation_type| -%>
+              self.<%= operation_type[:operation_name] %> = <%= operation_type[:operation_body] %>.new(client)
+              <%- end -%>
+              self.model_types = ModelTypes.new
+            end
+          end
+
+          class ModelTypes
+            <%- @model_types.each do |model_type| -%>
+            def <%= model_type[:method_name] %>
+              <%= model_type[:method_body] %>
+            end
+            <%- end -%>
+          end
+        end
+      end
+    end
+  end
+end
```

Comment:
please leave new line character at the end in template

Reply:
Done